### PR TITLE
Add using APP_ROOT_PATH and PLATFORM_PATH vars in install scripts

### PR DESCRIPTION
**Pull request by KovalM**

### CODE REVIEW

File: scripts/install_ostis.sh [hunk 1]
```diff
@@ -1,69 +1,73 @@
...
 	echo 'modules_path = ${SC_MACHINE_ROOT}/sc-kpm/sc-python/services;${SC_MACHINE_ROOT}/../../problem-solver/py/services' >> ./config.ini.in
-	cd ..
-	
-	cd ./scripts
+
+	cd ../scripts
 	./make_all.sh
-	cat ../bin/config.ini >> ../../config/sc-web.ini
-	cd ../..
+	cat ../bin/config.ini >> "${PLATFORM_PATH}"/config/sc-web.ini
```
Comment (ValikVityaz) [inline scripts/install_ostis.sh]: '..' is replaced everywhere but here. Shouldn't it be `cat "${PLATFORM_PATH}"/sc-machine/bin/config.ini`?

File: scripts/clone_subsystems.sh [hunk 2]
```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
-# Remove it after package manager implementation
+# Remove this script after package manager implementation
 
-cd ..
+APP_ROOT_PATH=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && cd .. && pwd)
+PLATFORM_PATH="${APP_ROOT_PATH}/ostis-web-platform"
+WORKING_PATH=$(pwd)
+
+cd ${APP_ROOT_PATH}
```
Comment (ValikVityaz) [inline scripts/clone_subsystems.sh]:
```suggestion
cd "${APP_ROOT_PATH}"
```
Should be wrapped in quotes everywhere.

File: scripts/clone_subsystems.sh [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
 if [ ! -d "./subsystems" ]
   then
     mkdir subsystems
```
Comment (ValikVityaz) [inline scripts/clone_subsystems.sh]:
```suggestion
then
	mkdir subsystems
```

File: scripts/clone_subsystems.sh [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
-    echo "../subsystems" >> ./ostis-web-platform/repo.path
+    echo "../subsystems" >> "${PLATFORM_PATH}"/repo.path
```
Comment (ValikVityaz) [inline scripts/clone_subsystems.sh]:
```suggestion
	echo "../subsystems" >> "${PLATFORM_PATH}"/repo.path
```

File: scripts/install_subsystems.sh [hunk 5]
```diff
@@ -1,12 +1,19 @@
...
-cd ../ostis-web-platform/sc-machine/scripts
+cd ${PLATFORM_PATH}/sc-machine/scripts
 ./make_all.sh
 
-cd ../../scripts
+cd ${PLATFORM_PATH}/scripts
 ./build_kb.sh
 
+cd ${WORKING_PATH}
+
```
Comment (ValikVityaz) [inline scripts/install_subsystems.sh]:
```suggestion

```

**Review by ValikVityaz — CHANGES_REQUESTED** [review]

**Review by MikhailSadovsky — APPROVED** [review]

File: scripts/install_ostis.sh [hunk 6]
```diff
@@ -1,69 +1,72 @@
...
-	cat ../bin/config.ini >> ../../config/sc-web.ini
-	cd ../..
+	cat "${PLATFORM_PATH}"/bin/config.ini >> "${PLATFORM_PATH}"/config/sc-web.ini
+	cd "${PLATFORM_PATH}"
 }
 
-cd ..
-if [ -d ${PLATFORM_FOLDER} ]; 
+cd "${APP_ROOT_PATH}"
+if [ -d ${PLATFORM_PATH} ];
```
Comment (ValikVityaz) [inline scripts/install_ostis.sh]:
```suggestion
if [ -d "${PLATFORM_PATH}" ];
```

**Review by ValikVityaz — APPROVED** [review]

**Pull request merged**